djangocms_snippet/templatetags/snippet_tags.py

```diff
@@ -0,0 +1,129 @@
+# -*- coding: utf-8 -*-
+
+"""
+Snippet template tags
+"""
+
+from contextlib import contextmanager
+
+from django import template
+from django.utils.safestring import mark_safe
+from django.utils.translation import ugettext_lazy as _
+
+from djangocms_snippet.models import Snippet
+
+register = template.Library()
+
+
+@contextmanager
+def exceptionless(truth):
+    # Accepts one truth parameter, when 'False' normal behavior
+    # when 'True' any expection will be suppressed
+    try:
+        yield
+    except Exception as e:
+        if truth:
+            # WARNING: suppresing expcetion
+            pass
+        else:
+            # Reraising exception
+            raise e
```

Comment:
I would we use catch ObjectDoesNotExist instead of all exceptions with Exception, mainly because I believe the idea is to just fallback to some content if the snippet does not exist which is the only operation being managed by this manager.

Also please change ``raise e`` to ``raise`` in order to avoid re-raising and losing some traceback info.